src/os/bluestore/BlueStore.cc

```diff
@@ -4169,7 +4169,7 @@ int BlueStore::_open_fm(bool create)
       bl.append(freelist_type);
       t->set(PREFIX_SUPER, "freelist_type", bl);
     }
-    fm->create(bdev->get_size(), t);
+    fm->create(bdev->get_size(), cct->_conf->bdev_block_size, t);
```

Comment:
should we send min_alloc_size? Anyways we are taking max of bdev_block_size, in freelist manager.